system/ee/ExpressionEngine/Tests/ExpressionEngine/Service/Formatter/TextFormatterTest.php

```diff
@@ -501,6 +501,9 @@ public function urlSlugProvider()
                 ],
                 'Sample-Title-to-Turn-Into-a-Slug-including-💩-tags-quotes-and-high-ascii-ssae-and-seps____in....content'
             ],
+            // ['ExpressionEngine®', [], 'expressionengine'],
```

Comment:
@intoeetive I added these tests to cover the issues described but this one won't pass, the formatter leaves the (R) symbol.  But when I test within the control panel it does get removed so I don't know how important it is to behave the same way and I'm not sure where the control panel behavior is defined (it seems like JS).

Reply:
@bryannielsen [®](https://www.htmlsymbols.xyz/unicode/U+00AE) is in our Emoji map, and we allow emojies in URL Slug. So I updated your unit tests script